File: 0001-two-sum/0001-two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    for (let x = 0; x < nums.length; x++)
    {
        for (let y = x+1 ; y<nums.length; y++)
        {
            if(nums[x]+nums[y]===target)
            {
               return[x,y];
            }
            else{
                continue;
            }
            
        }
    }
    return null;
};

File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        if(nums == null || nums.length == 0 ) return new int[0];
        Map<Integer, Integer> map = new HashMap<>();
        for(int i = 0; i <  nums.length; i++){
            int comp = target - nums[i];
            if(map.containsKey(comp)){
                return new int[]{map.get(comp), i};
            }
            map.put(nums[i], i);
            
        }
        return new int[0];
    }
}